QuowMinimap.lua

```diff
@@ -22923,7 +22926,7 @@ end
 -- Pick a font size and return it
 function QuowPickFontSize(sCurrentFontSize, sForWindowTitle)
   local iNewFontSize = 0
-  local sPopupResult = utils.inputbox("Please enter a custom font-size for the " .. sForWindowTitle .. " window.\n\nNote: Not all fonts support all sizes, try between 8 to 16.\n", sForWindowTitle .. " Font Size", sCurrentFontSize, "", 12 )
+  local sPopupResult = utils.inputbox("Please enter a custom font-size for the " .. sForWindowTitle .. " window.\n\nNote: Not all fonts support all sizes, try between 8 to 16.\n", sForWindowTitle .. " Font Size", tostring(sCurrentFontSize), "", 12 )
```

Comment:
SmushClient's API uses strict typechecking. If a function expects a string, it will throw an error if it receives something else, rather than silently converting it to a string.